### ISSUE TITLE
Dead Keys not working in qutebrowser

### ISSUE THREAD

**Issue by schyzophrene-asynchrone** [qutebrowser/qutebrowser#4999]
**Version info (see `:version`)**:
```qutebrowser v1.7.0
Git commit: 
Backend: QtWebEngine (Chromium 73.0.3683.105)

CPython: 3.7.4
Qt: 5.13.0
PyQt: 5.13.0

sip: 4.19.18
colorama: 0.4.1
pypeg2: 2.15
jinja2: 2.10.1
pygments: 2.4.2
yaml: 5.1.2
cssutils: no
attr: 19.1.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.29.0
QtNetwork SSL: OpenSSL 1.1.1c  28 May 2019

Style: QFusionStyle
Platform: Linux-5.2.11-arch1-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.7/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt
```

**Does the bug happen if you start with `--temp-basedir`?**:
Yes.
**Description**
I am using a bepo keyboard layout (bepo.fr) which uses dead keys to allow input of some caracters (ë for instance). However, qutebrowser seems to ignore dead keys, as I cant type any of them. It may be a bug there for quite a while, as the first time I probably encountered it was when typing some password, though I did not understand there what was the bug. The dead keys are correctly understood by other applications (urxvt, firefox as I essentialy use these).
This is true in command mode as in input mode.
**How to reproduce**
`setxkbmap -layout fr -variant bepo` and then AltGr+i followed by u should input ü but in qutebrowser it inputs u.

**Comment by arza-zara** [comment]
Works for me with similar setup. Try with [Falkon](https://www.falkon.org/) to see if it's Qt-specific.
<details><summary>Version info</summary>

```
qutebrowser v1.7.0
Git commit: f2b38a6c2 (2019-09-02 23:19:53 +0200)
Backend: QtWebEngine (Chromium 73.0.3683.105)

CPython: 3.7.4
Qt: 5.13.0
PyQt: 5.13.0

sip: 4.19.18
colorama: 0.4.1
pypeg2: 2.15
jinja2: 2.10.1
pygments: 2.2.0
yaml: 5.1.2
cssutils: 1.0.2 $Id$
attr: 19.1.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: 2.2.228 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.29.0
QtNetwork SSL: OpenSSL 1.1.1c  28 May 2019

Style: QFusionStyle
Platform: Linux-5.0.12-arch2-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /home/user/ohjelmat/qutebrowser-git/src/qutebrowser/qutebrowser
Using Python from /usr/bin/python
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt
```
</details>

**Comment by schyzophrene-asynchrone** [comment]
It’s not working in Falkon either. Probably a Qt bug then. Could someone report it ? I’m not sure where I should do that…

**Comment by arza-zara** [comment]
It might already be fixed in Qt 5.13.1 ([QTBUG-77041](https://bugreports.qt.io/browse/QTBUG-77041)). Help for reporting: [Reporting bugs](https://wiki.qt.io/Reporting_Bugs).

**Comment by The-Compiler** [comment]
Works fine for me as well (note this is <kbd>AltGr+d</kbd>, <kbd>s</kbd> on a qwerty keyboard) - on Archlinux with the same Qt version.

Not sure what's different about your setup @schyzophrene-asynchrone, but I can't really report a bug I can't reproduce myself. Since it works for other people, Qt developers probably won't be able to do anything without more information either.

@arza-zara That bug is only for the wasm platform plugin, i.e. running Qt inside a browser via WebAssembly.

Closing this as it's not a qutebrowser bug - sorry I can't do more! If you find out how I could reproduce it, I'd be happy to open an upstream issue though.

**Comment by schyzophrene-asynchrone** [comment]
I found out where the bug came from : my LANG was set to de_DE.UTF8. But there were no such entry in /usr/share/X11/locale/compose.dir so I changed my locale to de_DE.UTF-8, rebooted, and now everything seems to be working fine.

I’m not sure why, though, since `locale` still says `LANG=de_DE.UTF8` and I’m running a wayland session anyway (so I don’t see what the compose file for X11 has to do with me), but still, it’s working, so I’m not going to complain.

Just putting it here in case it may help someone else.
